extensions/extensions-docker/aissemble-hive/aissemble-hive-service/pom.xml

```diff
@@ -15,4 +15,57 @@
 
     <packaging>orphedomos</packaging>
 
+    <properties>
+        <dockerbuild.jars.directory>target/dockerbuild/jars/</dockerbuild.jars.directory>
+        <version.hive.metastore>4.0.0</version.hive.metastore>
```

Comment:
I: Managing dependencies through Maven as a matter of good practice and centralization.